.github/workflows/benchmarking-runner.yml

```diff
@@ -0,0 +1,36 @@
+name: 'Benchmarking Runner'
+
+on: # rebuild any PRs and main branch changes
+  pull_request:
+  push:
+    branches:
+      - main
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+
+      - name: Checkout (GitHub)
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ github.head_ref }}
+
+      - name: Set up Git
+        run: |
+          git config --local user.email "github-actions[bot]@users.noreply.github.com"
+          git config --local user.name "github-actions[bot]"
+
+      - name: Build and run Dev Container task
+        uses: devcontainers/ci@v0.3
+        with:
+          imageName: graphql-benchmarks
+          push: never
+          runCmd: |
+            bash ./setup.sh
+            bash ./run_benchmarks.sh
+
+      - name: Push changes
+        uses: ad-m/github-push-action@master
+        with:
+          branch: ${{ github.head_ref }}
```

Comment:
Use the runner below to make the results are consistent.

<img width="1386" alt="Screenshot 2023-11-17 at 1 25 16 PM" src="https://github.com/tailcallhq/graphql-benchmarks/assets/194482/4265bd3c-6d5f-46be-bcdb-ca84c4eab717">